build.sh

```diff
@@ -410,7 +464,18 @@ fetch_libkalibera() {
     fi
 }
 
+
 fetch_external_benchmarks
+build_gcc
+
+# Put GCC libs into linker path
+# Needed for (e.g.) V8 to find libstdc++
+if [ `uname` = "Linux" ]; then
```

Comment:
This should be a `case ... esac` so that we can easily add other OSs later (the default branch should probably error?).